test/functional/services/remote/network_profiles.ts

```diff
@@ -12,23 +12,11 @@ interface NetworkOptions {
   LATENCY: number;
 }
 
-const sec = 1_000;
-const kB = 1024;
+const sec = 10 ** 3;
+const MBps = 10 ** 6 / 8; // megabyte per second (MB/s) (can be abbreviated as MBps)
 
-// Download (kb/s)	Upload (kb/s)	Latency (ms)
-// https://gist.github.com/theodorosploumis/fd4086ee58369b68aea6b0782dc96a2e
+// Selenium uses B/s (bytes) for network throttling
+// Download (B/s)	Upload (B/s) Latency (ms)
 export const NETWORK_PROFILES: { [key: string]: NetworkOptions } = {
-  DEFAULT: { DOWNLOAD: 5 * kB * sec, UPLOAD: 1 * kB * sec, LATENCY: 0.1 * sec },
-  GPRS: { DOWNLOAD: 0.05 * kB * sec, UPLOAD: 0.02 * kB * sec, LATENCY: 0.5 * sec },
-  MOBILE_EDGE: { DOWNLOAD: 0.24 * kB * sec, UPLOAD: 0.2 * kB * sec, LATENCY: 0.84 * sec },
-  '2G_REGULAR': { DOWNLOAD: 0.25 * kB * sec, UPLOAD: 0.05 * kB * sec, LATENCY: 0.3 * sec },
-  '2G_GOOD': { DOWNLOAD: 0.45 * kB * sec, UPLOAD: 0.15 * kB * sec, LATENCY: 0.15 * sec },
-  '3G_SLOW': { DOWNLOAD: 0.78 * kB * sec, UPLOAD: 0.33 * kB * sec, LATENCY: 0.2 * sec },
-  '3G_REGULAR': { DOWNLOAD: 0.75 * kB * sec, UPLOAD: 0.25 * kB * sec, LATENCY: 0.1 * sec },
-  '3G_GOOD': { DOWNLOAD: 1.5 * kB * sec, UPLOAD: 0.75 * kB * sec, LATENCY: 0.04 * sec },
-  '4G_REGULAR': { DOWNLOAD: 4 * kB * sec, UPLOAD: 3 * kB * sec, LATENCY: 0.02 * sec },
-  DSL: { DOWNLOAD: 2 * kB * sec, UPLOAD: 1 * kB * sec, LATENCY: 0.005 * sec },
-  CABLE_5MBPS: { DOWNLOAD: 5 * kB * sec, UPLOAD: 1 * kB * sec, LATENCY: 0.28 * sec },
-  CABLE_8MBPS: { DOWNLOAD: 8 * kB * sec, UPLOAD: 2 * kB * sec, LATENCY: 0.1 * sec },
-  WIFI: { DOWNLOAD: 30 * kB * sec, UPLOAD: 15 * kB * sec, LATENCY: 0.002 * sec },
+  CLOUD_USER: { DOWNLOAD: 6 * MBps, UPLOAD: 6 * MBps, LATENCY: 0.1 * sec },
```

Comment:
Anyone know why we removed these? I was looking for a way to throttle during a functional test to confirm a loading theory. Seems like this might have worked and I was about to add `3G_SLOW` back. But maybe I shouldn't?

Reply:
We removed them because they were inaccurate and finding information that accurately describes the speeds used by the browser profiles was really hard, so we punted on it.

Reply:
Gotcha. https://github.com/elastic/kibana/pull/135103 has the snippet I ended up using instead (to slow a test down enough to fail). Maybe this sort of thing would be better as a helper rather than webdriver-init-time setting 🤔